### ISSUE TITLE
iOS Compatibility Issues: Errors with AdaptiveButton, AdaptiveNavigationBarItem, and AdaptiveSwitch on compose-multiplatform 1.7.0

### ISSUE THREAD

**Issue by alxmtzr** [alexzhirkevich/compose-cupertino#71]
### iOS Compatibility Issues: Errors with `AdaptiveButton`, `AdaptiveNavigationBarItem`, and `AdaptiveSwitch`

Hi, I recently started working with Compose Multiplatform and encountered issues on iOS with certain Adaptive components. When using `AdaptiveButton`, `AdaptiveNavigationBarItem`, and `AdaptiveSwitch`, I’m seeing `IrLinkageError` exceptions due to Android-specific functions (`clickable`, `selectable`, and `toggleable`) that aren't available on iOS.

#### Errors Encountered:
- **`AdaptiveButton`**: `Function 'clickable' cannot be called`
- **`AdaptiveNavigationBarItem`**: `Function 'selectable' cannot be called`
- **`AdaptiveSwitch`**: `Function 'toggleable' cannot be called`

**Example stack trace for `clickable` error**:

```
Uncaught Kotlin exception: kotlin.native.internal.IrLinkageError: Function 'clickable' can not be called: No function found for symbol 'androidx.compose.foundation/clickable|clickable@androidx.compose.ui.Modifier(androidx.compose.foundation.interaction.MutableInteractionSource;androidx.compose.foundation.Indication?;kotlin.Boolean;kotlin.String?;androidx.compose.ui.semantics.Role?;kotlin.Function0<kotlin.Unit>){}[0]'
```

I'm not sure if I did something wrong.

Thank you!

**Comment by begali** [comment]
Duplicate of https://github.com/alexzhirkevich/compose-cupertino/issues/55

**Comment by alexzhirkevich** [comment]
It was a Compose bug fixed in [1.7.1](https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.7.1)
